src/gui/dialogues/newpilotdialog.cpp

```diff
@@ -167,18 +167,20 @@ void NewPilotDialog::submitForm()
     newData.insert("displayname",displayName);
 
     using namespace experimental;
-    auto uin = newPilotInput(newData);
+    //auto uin = newPilotInput(newData);
+    auto entry = PilotEntry(newData);
+
+    if (role == Db::editExisting)
+        //entry.setPosition = oldEntry.position; //needs implementation in Entry class
```

Comment:
If `editExisting` means that the user clicked an existing entry and poisition is pre-filled. No need for setPosition.

Reply:
adjusted.